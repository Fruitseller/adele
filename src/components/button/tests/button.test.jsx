import React from 'react';
import 'jest-styled-components';

import Button from '../button';

const mockFunc = () => true;

it('renders filled Button correctly', () => {
  const wrapper = mount(<Button label="String" link="String" type="filled" targetBlank="_blank" />);
  expect(wrapper).toMatchSnapshot();
});

it('renders ghost Button correctly', () => {
  const wrapper = mount(<Button label="String" link="String" type="ghost" targetBlank="_blank" />);
  expect(wrapper).toMatchSnapshot();
});
