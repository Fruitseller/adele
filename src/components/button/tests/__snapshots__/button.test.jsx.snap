// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders filled Button correctly 1`] = `
<Button
  label="String"
  link="String"
  tab={0}
  targetBlank="_blank"
  type="filled"
/>
`;

exports[`renders ghost Button correctly 1`] = `
<Button
  label="String"
  link="String"
  tab={0}
  targetBlank="_blank"
  type="ghost"
/>
`;
